README.md

```diff
@@ -6,7 +6,7 @@ The modern web framework
 for Java
 
 ![Latest Stable Version](https://img.shields.io/npm/v/@hilla/frontend.svg)
-[![Releases](https://img.shields.io/badge/maven-v1.0.3-1d77e4)](https://github.com/vaadin/hilla/releases)
+[![Releases](https://img.shields.io/badge/maven-v1.0.4-1d77e4)](https://github.com/vaadin/hilla/releases)
```

Comment:
```suggestion
[![Releases](https://img.shields.io/maven-metadata/v?metadataUrl=https%3A%2F%2Frepo1.maven.org%2Fmaven2%2Fdev%2Fhilla%2Fhilla%2Fmaven-metadata.xml)](https://github.com/vaadin/hilla/releases)
```

should we make it automatic? It gets a bit repetitive to update the README every time.

Reply:
Ah.that is great.  I didn’t know It works this way.

Reply:
Updated